var/spack/repos/builtin/packages/mpich/package.py

```diff
@@ -90,6 +90,9 @@ class Mpich(AutotoolsPackage):
 '-Wl,-flat_namespace'.'''
     )
 
+    variant('max_vcis', values=int, default=1, when='netmod=ucx',
+            description='Select max number of VCIs to configure')
```

Comment:
VCIs work with both the ofi and ucx netmods in ch4. And VCIs do not work with ch3, so this variant I guess should be limited to `when='device=ch4'`?

Reply:
Indeed, it is a bad copy/paste when i refactored my code. Already resolved.